chapters/unitexpressions.tex

```diff
@@ -59,20 +59,26 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
 
 unit-symbol :
    unit-char { unit-char }
+
+unit-char :
+   NON-DIGIT
 \end{lstlisting}
 
 A \lstinline[language=grammar]!unit-symbol! is a non-empty string of \lstinline[language=grammar]!unit-char! characters.
```

Comment:
Sorry, my suggestion was to also remove this sentence.  It is redundant now.

Reply:
Ok!